odata-protocol/11.4 Data Modification.md

```diff
@@ -178,10 +178,18 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Non-insertable properties SHOULD be omitted from the request body.
-If they are provided, services MUST either ignore the values in the request body or fail the request
+If non-insertable properties are included in the request body,
+services MUST either ignore them or fail the request
 if the provided values do not match the service-determined values.
 
+Assume a client has out-of-band knowledge that a service has the latter behavior for
```

Comment:
This could be made explicit if such properties were listed in `Capabilities.InsertRestrictions/PostconditionProperties` instead of (or in addition to) `NonInsertableProperties`.